### PR TITLE
Implementation of missing ISI execute methods and test.

### PR DESCRIPTION
**Pull request by humb1t**
Signed-off-by: Nikita <humb1t@yandex.ru>

### CODE REVIEW

File: iroha_data_model/src/isi.rs [hunk 1]
```diff
@@ -141,12 +141,12 @@ where
 
 /// Generic instruction for a removal of an object from the identifiable destination.
 #[derive(Debug, Clone, Serialize, Deserialize, Encode, Decode)]
-pub struct Remove<D, O>
+pub struct Substract<D, O>
```
Comment (lebdron) [inline iroha_data_model/src/isi.rs:144]:
```suggestion
pub struct Subtract<D, O>
```

File: iroha_data_model/src/isi.rs [hunk 2]
```diff
@@ -141,12 +141,12 @@ where
...
 where
     D: Identifiable,
     O: Value,
 {
-    /// Object which should be removed.
+    /// Object which should be substractd.
```
Comment (lebdron) [inline iroha_data_model/src/isi.rs:149]:
```suggestion
    /// Object which should be subtracted.
```
Please check this throughout the PR

File: iroha_data_model/src/isi.rs (hunk 1 above)
Comment (humb1t) [reply iroha_data_model/src/isi.rs:144]: @lebdron - thanks, regex replacement was not a great idea)))

**Review by eadventurous — APPROVED** [review]
Apart from `Subtract` naming mistake, everything else looks good

**Pull request merged**